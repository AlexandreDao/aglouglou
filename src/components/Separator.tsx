import { View, StyleSheet } from 'react-native'
import React from 'react'

const styles = StyleSheet.create({
  separator: {
    height: 10,
  },
})

const Separator = () => {
  return <View style={styles.separator} />
}

export default Separator
