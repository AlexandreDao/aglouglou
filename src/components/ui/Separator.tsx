import { View, StyleSheet } from 'react-native'
import React from 'react'

const styles = StyleSheet.create({
  separator: {
    height: 10,
  },
})

function Separator() {
  return <View style={styles.separator} />
}

export default Separator
