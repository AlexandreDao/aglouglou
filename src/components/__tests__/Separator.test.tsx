import React from 'react'
import { render } from '@testing-library/react-native'
import Separator from '@/components/ui/Separator'

describe('<Separator />', () => {
  test('Separator snapshot', () => {
    const tree = render(<Separator />).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
