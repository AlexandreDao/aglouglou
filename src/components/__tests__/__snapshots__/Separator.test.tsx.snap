// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator /> Separator snapshot 1`] = `
<View
  style={
    {
      "height": 10,
    }
  }
/>
`;
